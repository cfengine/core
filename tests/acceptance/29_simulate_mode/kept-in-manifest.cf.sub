body common control
{
  inputs => { "../default.cf.sub" };
}

bundle agent main
{
  files:
    "$(G.testroot)$(const.dirsep)already-created"
      create => "true";
}
