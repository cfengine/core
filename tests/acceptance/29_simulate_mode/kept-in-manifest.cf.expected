 warning: All changes in files will be made in the 'WORKDIR/state/PID.changes' chroot
===========================================================================
'WORKDIR/tmp/already-created' is a regular file
Size: SIZE
Access: (0600/rw-------)  Uid: (0/root)   Gid: (0/root)
Access: TIMESTAMP
Modify: TIMESTAMP
Change: TIMESTAMP

Contents of the file:
